authz/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id "java-library"
+    id "maven-publish"
+
+    id "com.github.johnrengelman.shadow"
+    id "com.google.protobuf"
+    id "ru.vyarus.animalsniffer"
+}
+
+description = "gRPC: Authorization"
+
+evaluationDependsOn(project(':grpc-core').path)
+
+dependencies {
+    implementation project(':grpc-protobuf'),
+            project(':grpc-core')
+
+    testImplementation project(':grpc-core').sourceSets.test.output
```

Comment:
Looks like you don't use any of the testing classes is grpc-core, so you can delete this and the `evaluationDependsOn()` above.

Reply:
Done